### PR TITLE
Completed assigned iteration 2 methods

### PR DESCRIPTION
**Pull request by whitneykidd**
Complete methods with passing tests for:  highest_total_score, lowest_total_score and biggest_blowout

### CODE REVIEW

File: lib/game_stats.rb [hunk 1]
```diff
@@ -8,4 +8,16 @@ class GameStats
   def initialize(file_path, object)
     @games = csv_data('./data/games_truncated.csv', Game)
   end
+
+  def highest_total_score
```
Comment (tylertomlinson) [inline lib/game_stats.rb:12]: That single line though

File: lib/game_stats.rb [hunk 2]
```diff
@@ -8,4 +8,16 @@ class GameStats
...
+    @games.map {|game| game.away_goals + game.home_goals}.max
+  end
+
+  def lowest_total_score
+    @games.map {|game| game.away_goals + game.home_goals}.min
+  end
+
+  def biggest_blowout
+    @games.map {|game| (game.away_goals - game.home_goals).abs}.max
```
Comment (tylertomlinson) [inline lib/game_stats.rb]: Nice work on using abs within the map

File: lib/game_stats.rb [hunk 3]
```diff
@@ -8,4 +8,16 @@ class GameStats
   def initialize(file_path, object)
     @games = csv_data('./data/games_truncated.csv', Game)
   end
+
+  def highest_total_score
+    @games.map {|game| game.away_goals + game.home_goals}.max
```
Comment (alex-latham) [inline lib/game_stats.rb]: Ruby style guide prefers a space before and after curly brackets. E.g., `@games.map { |game| game.away_goals + game.home_goals }.max`

File: test/game_stats_test.rb [hunk 4]
```diff
@@ -27,4 +27,16 @@ def test_attributes_for_instance_of_game_withing_game_stats
     assert_equal 2  , @game.away_goals
```
Comment (alex-latham) [inline test/game_stats_test.rb]: Unusual spacing after 2

File: lib/game_stats.rb [hunk 5]
```diff
@@ -8,4 +8,16 @@ class GameStats
...
+  end
+
+  def lowest_total_score
+    @games.map {|game| game.away_goals + game.home_goals}.min
```
Comment (alex-latham) [inline lib/game_stats.rb]: Love the single-line syntax

File: lib/game_stats.rb (hunk 3 above)
Comment (Maxwell-Baird) [reply lib/game_stats.rb]: Good job on saving line space by putting max at the end

**Pull request merged**